AsyncDisplayKit/Layout/ASStackLayoutSpec.h

```diff
@@ -48,6 +48,12 @@
 
 - (instancetype)init;
 
+/** Specifies how children are aligned horizontally. The stack direction will become immutable after this call. */
+- (void)setHorizontalAlignment:(ASHorizontalAlignment)alignment;
```

Comment:
These should be a property, if possible.  Although it is not likely to be a common use case to read the value, I think the same is true of alignItems / justifyContent, and the stylistic ability to use the property syntax to set the value will help the initialization of stacks look neater.

We should format the options near, but above, the alignItems / justifyContent properties and try to make it visually (and in description) clear that 1) these options are generally preferred to the others, and 2) if used, they supersede / replace the others, and 3) setting either of them will impact the other.

Implementing that 3) aspect would increase the complexity of this, so I am actually happy to land this diff without that change, but wanted to call it out as critical for the final 2.0.  @rcancro can do that piece if you're not available, @nguyenhuy - just let us know!